### PR TITLE
Updated Sign and Verify Api calls for work with strings

### PR DESCRIPTION
**Pull request by Artemkaaas**
Added Ledger SignRequest Api call

### CODE REVIEW

File: libindy/include/indy_ledger.h [hunk 1]
```diff
@@ -68,6 +68,39 @@ extern "C" {
...
+    /// #Returns
+    /// Signed request json.
+    ///
+    /// #Errors
+    /// Common*
+    /// Wallet*
+    /// Ledger*
+    /// Crypto*
+
+    extern indy_error_t indy_sign_and_submit_request(indy_handle_t command_handle,
```
Comment (jovfer) [inline libindy/include/indy_ledger.h]: Is it indy_sign_request?

File: libindy/include/indy_ledger.h [hunk 2]
```diff
@@ -68,6 +68,39 @@ extern "C" {
...
+                                                         indy_handle_t wallet_handle,
+                                                         const char *    submitter_did,
+                                                         const char *    request_json,
+
+                                                         void           (*cb)(indy_handle_t xcommand_handle,
+                                                                              indy_error_t  err,
+                                                                              const char*     signed_request_json)
+                                                         );
+
+
```
Comment (jovfer) [inline libindy/include/indy_ledger.h]: @vimmerru should we also implement indy_verify_request?

Comment (vimmerru) [inline libindy/include/indy_ledger.h]: No, I don't see a use case for this request. We sign request to be only verified by pool nodes.

File: ci/java.dockerfile [hunk 3]
```diff
@@ -11,7 +11,7 @@ RUN apt-get install -y \
       gdebi \
       apt-utils
 
-ADD https://repo.evernym.com/deb/indy-sdk/0.1.1-100/indy-sdk_0.1.1_amd64.deb .
+ADD https://repo.evernym.com/deb/indy-sdk/0.1.1-7/indy-sdk_0.1.1_amd64.deb .
```
Comment (jovfer) [inline ci/java.dockerfile]: Should not be merged into master

File: libindy/include/indy_signus.h [hunk 4]
```diff
@@ -165,7 +166,8 @@ extern "C" {
                                                   indy_handle_t pool_handle,
                                                   
                                                   const char *    did,
-                                                  const char *    signed_msg,
+                                                  const char *    msg,
```
Comment (jovfer) [inline libindy/include/indy_signus.h]: Should operate with byte arrays, not with strings.

**Pull request merged**